### PR TITLE
Update all network configs - Closes #668

### LINKED ISSUE #668: Update all network configs
### Description

Update the config files to the latest format.

### Acceptance Criteria
- The `config.json` for the following networks is up-to-date and adheres to the latest format.
  - `betanet`
  - `testnet`
  - `mainnet`
- The `chainID` is updated correctly for all the networks including `devnet` and `alphanet`
- Add a placeholder for supported tokens in the config.

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #668

### How was it solved?

- [x] The config.json for the following networks is up-to-date and adheres to the latest format.
  - [x] betanet
  - [x] testnet
  - [x] mainnet
- [x] The chainID is updated correctly for all the networks including devnet and alphanet
- [x] Add a placeholder for supported tokens in the config (can be handled with the issue https://github.com/LiskHQ/lisk-core/issues/656 as it is not finalized).

### How was it tested?
Local

### CODE REVIEW

File: config/betanet/config.json [hunk 1]
```diff
@@ -1,34 +1,25 @@
...
-	"logger": {
-		"fileLogLevel": "error",
-		"consoleLogLevel": "info"
+	"generator": {
+		"keys": {
+			"fromFile": "./config/dev-validators.json"
+		}
 	},
 	"network": {
+		"version": "1.0",
```
Comment (sameersubudhi) [inline config/betanet/config.json:22]: Current network version in production is `3.1` (Ref: https://service.lisk.com/api/v2/network/status).
Could you please confirm with the SDK/Research team what should be the value for the network version? Probably, `4.0`?

Comment (ManuGowda) [reply config/betanet/config.json:22]: @sameersubudhi @nagdahimanshu this is ok, for dev/alpha/beta the network version will be 1.0.
unless we migrate the supported betanet with new minor/major version, which we don't do, we create new betanet.

so 1.0 is correct.

**Review by ManuGowda — APPROVED** [review]

File: config/betanet/config.json (hunk 1 above)
Comment (sameersubudhi) [reply config/betanet/config.json:22]: @ManuGowda We'll maintain the network version to be `1.0` in the devnet, alphanet and betanet configs. Then, should we update the testnet and mainnet network version to `4.0`?

Comment (sameersubudhi) [reply config/betanet/config.json:22]: @nagdahimanshu I clarified with @ManuGowda offline, we can update the network version to `4.0` for the testnet and mainnet configs.

**Pull request merged**